src/ucs/sys/math.h

```diff
@@ -24,18 +24,18 @@ BEGIN_C_DECLS
 #define UCS_GBYTE    (1ull << 30)
 #define UCS_TBYTE    (1ull << 40)
 
-#define ucs_min(a, b) \
+#define ucs_min(_a, _b) \
```

Comment:
fixing macro args in math.h is unrelated fix which should be in separate PR

Reply:
#3294 covers this